docs/Release_Notes/v2024.1.0.md

```diff
@@ -0,0 +1,39 @@
+# v2024.1.0
+
+Previous version: [v2023.34.0](v2023.34.0.md)
+
+## Features
+
+### Interface of a DIC template
+ Implemented new user interface for managing device event templates.
+
+## Enhancements
+
+### New device groups and assigned devices
+Added a possibility to create a new group when assigning devices.
+
+### Block size configuration
+Added a possibility to configure the block size in CoAP Block transfer communication for a device or group.
+
+### Disabling a handlers
+Added a checkbox `Keep active after repeated fails` that enables restoring the connection automatically when endpoint is available again. Available for users with administrative permissions.
+
+### Hide passwords and tokens
+Passwords and tokens for basic authorization are now hidden behind asterisks in the authorization method view.
+
+### Influx templates
+Updated the default Influx telemetry template.
+
+## Bug Fixes
+
+### Provisioning session error
+Fixed the provisioning session error showing a null value in database.
+
+### Read resource during the FOTA update
+Fixed a bug causing an incorrect state of unified devices during the FOTA update being displayed.
```

Comment:
From the user perspective it is more about something like that: Fixed a bug causing an incorrect error message displayed when task was executed during the FOTA update.